src/api/endpoints.ts

```diff
@@ -25,6 +25,10 @@ export const endpoints = {
     my: '/pipelines/',
     get: (pipelineId: TId): string => `/pipelines/${pipelineId}`,
   },
+  Stacks: {
+    my: '/Stacks/',
+    get: (pipelineId: TId): string => `/Stacks/${pipelineId}`,
```

Comment:
Shouldn't this be stackId rather than pipelineId

Reply:
fixed